build.sbt

```diff
@@ -39,8 +39,8 @@ libraryDependencies ++= Seq(
   commonsIO,
   guava,
   scalaz,
-  junit4,
-  junitInterface,
+  junit5,
+  "net.aichler" % "jupiter-interface" % JupiterKeys.jupiterVersion.value % Test,
```

Comment:
What is this?

Reply:
ah! thx!
junit5 - ok stands for junit5 dependency
the next line is updated version of the dependency that previously lived inside `junitInterface` variable
this "interface" give access to test trough sbt command line. So you can type `sbt test` or `sbt testOnly @SomeTest` this is crucial for our CI
I will wrap it back to the `junitInterface` var in a minute